web/client-api/src/main/java/io/deephaven/web/client/api/subscription/scratch

```diff
@@ -0,0 +1,13 @@
+Trigger Select Distinct
```

Comment:
Extraneous file, yeah? If it's useful for reviewer to see, might post as comment.

Reply:
Whoops, yea I did not mean to include that. The race condition can be identified a little more clearly than mentioned in the doc, I'll update the description.